lms/db/__init__.py

```diff
@@ -1,5 +1,7 @@
 import logging
 
+import alembic.command
+import alembic.config
```

Comment:
`import alembic` doesn't work, you have to do it like this

Reply:
Is this because we're trying to reach one extra down the hierarchy (alembic > command > stamp) or does this just not work full stop?

Reply:
I dunno:

```python
>>> import alembic
>>> alembic.command
AttributeError
```

It's always done as `import alembic.command`

:man_shrugging: Not worth worrying about